bindings/python/tests/conftest.py

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+
+import opendal
+import pytest
+
+
+@pytest.fixture
+def operator() -> opendal.Operator:
+    # Get current service name from envs.
+    service_name = os.environ.get('TEST_SERVICE_NAME')
```

Comment:
It's better to support running test for multiple services at the same time.

Reply:
I'm not sure how to implement that, a group of tests usually relies on a specialized external service in this scene, maybe we should run multiple groups of tests in different containers at the same time?

Reply:
Adding a env like TEST_SERVICE_NANE makes it hard to maintain and use. Is it possible to check OPENDAL_XXX_TEST only? So that we can reuse the same env settings for all components of opendal.

Reply:
I have considered this solution, and we need to make sure that only one TEST is marked on at a time, because our fixture only initializes one operator.

It may cause confusion when multiple TESTs are marked as on.

If this is not a problem, I will update the code according to this solution.

Reply:
@Xuanwo I updated my implementation, could you please help me review it?